src/types/profile.js

```diff
@@ -184,6 +191,7 @@ export type ProfileMeta = {|
   oscpu: string,
   platform: string,
   processType: number, // TODO find the possible values
+  extensions: ExtensionTable,
```

Comment:
same here

Reply:
They should probably both be marked as optional, yeah.